### DISCUSSION TITLE
Can i undo a face merge?

### DISCUSSION THREAD

**Discussion by t3dium** [immich-app/immich#4111] [Q&A] [answered]
I accidentally merged two faces together, is it possible to undo the merge? Or to reset all face training data.

**Comment by bo0tzz** [comment]
Undoing a merge is not possible, you can reset all the face data by rerunning the face recognition job for all assets.

**Reply by t3dium** [reply]
I'm assuming by resetting the face data, that would also undo the merges i did right?

**Reply by bo0tzz** [reply, marked as answer]
Yes, indeed

**Reply by mjschrod** [reply]
I hope my question is not too lame, but how do I reset the face data resp. rerun the recognition job?
